modules/standard/Regex.chpl

```diff
@@ -1023,18 +1092,11 @@ operator regex.=(ref ret:regex(?t), x:regex(t))
     }
     ret._regex = x._regex;
```

Comment:
Do we not need this regex assignment to occur inside of the on-stmt?

Reply:
Good question. It does not need to be, since it is just copying the pointer over. I've tested with inside the `on` and it also passes. Trying to think of other considerations for whether it matters one way or the other

Reply:
Ah, good point -- does it add an extra GET/PUT if it's not within the on?